### ISSUE TITLE
topy identifies many many typos in twisted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7252]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7252|
|Type|defect|
|Created|2014-04-25 18:00:20Z|
|Branch|https://github.com/twisted/twisted/tree/fix-many-typos-7252|

I've audited this list for real ones.

Attachments:

* [typo-patch-cleanup.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/70a/70af46c040bd66c4afc5f194679ab76dd9b74572/134b8c255f0c5def0c9f29c11ef97dd814d1ab78.diff) (1462 bytes) - added by Alex on 2014-10-15 04:53:16Z - Fixes for issues #4 and #5

<details><summary>Searchable metadata</summary>

```
trac-id__7252 7252
type__defect defect
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__branches_fix_many_typos_7252 branches/fix-many-typos-7252
branch_author__dreid dreid
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1398448820000000 1398448820000000
changetime__1413349404909555 1413349404909555
version__None None
owner__habnabit habnabit

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

https://github.com/twisted/twisted/pull/25/files resolves these

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Continuous integration?

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Alex commented|
|-|-|

Sorry, I'm not sure I understand the question.

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)<a name="note_4"></a>|@dreid commented|
|-|-|

(In [42551]) Branching to fix-many-typos-7252.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|Alex commented|
|-|-|

Branch has passing tests: https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/fix-many-typos-7252

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban commented|
|-|-|

I have checked the patch and it looks good.

Missing news file!

Maybe the ticket summary should be "Fix typos from ALL codebase."

I am not sure how constructive this is without a CI check to keep the codebase typo free but I have no idea of a sane typo linter.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)|@habnabit set owner to @habnabit|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_8"></a>|@habnabit set owner to Alex|
|-|-|

I had to re-force a build as there were no results in the above buildbot link.

A couple of minor things:

1. twisted/test/test_defgen.py: Is `L{twisted.internet.defer.Deferred}` required? I thought you could write just `L{Deferred}`. Not a big deal, though.
1. twisted/lore/test/test_lore.py: Why are lines 73-74 merged into one line? I'm surprised twistedchecker didn't complain.
1. twisted/conch/ssh/connection.py: I noticed line 342 says "Called if the other wisde wanted a reply to the channel requeset and"—would you mind fixing those typos too?
1. docs/lore/howto/lore.rst: Removing an extra 'for' broke the table; it's missing whitespace before the trailing `|` now.
1. No topfile! Please add one.

Of these, #4 and #5 are the only serious issues. Fix those, at the very least, and this is good to merge.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_10"></a>|Alex set owner to @habnabit|
|-|-|

Issues #4 and #5 are resolved by that patch.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_11"></a>|@habnabit commented|
|-|-|

Thank you! Looks good now; going to merge this.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_12"></a>|@habnabit commented|
|-|-|

(In [43372]) Apply patch from Alex. refs [#7252](https://github.com/twisted/twisted/issues/7252)

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_13"></a>|@habnabit set status to `closed`|
|-|-|

(In [43373]) Merge fix-many-typos-7252: Fix many typos in Twisted.

Authors: Alex, dreid
Reviewer: habnabit
Fixes: [#7252](https://github.com/twisted/twisted/issues/7252)

There were many typos present in Twisted documentation (sphinx and docstrings
both) and comments; they have now been fixed.
